### ISSUE TITLE
Breakout Example

### ISSUE THREAD

**Issue by kmader** [karpathy/reinforcejs#15]
In case it's interesting or you have a sample gallery, breakout with deep-q learning (since that's the youtube video example originally shown for such games)
http://4quant.com/javascript-breakout/ repo: https://github.com/4Quant/javascript-breakout

**Comment by gitricko** [comment]
Yes, second that... I probably, will take a stab at that to see if we can integrate ReinforceJS to this js-breakout game. 

Another question: Does the javascript version able to leverage the GPU ?

**Comment by gitricko** [comment]
@kmader Just got a simple breakout game working using a simple code from [MDN](https://developer.mozilla.org/en-US/docs/Games/Tutorials/2D_Breakout_game_pure_JavaScript)

Actually, before trying out breakout, i was trying reinforcejs on "snake eat apple game" and got some success. Thinking it is easy, i move on to breakout but never got it work properly. I keep changing the game settings like using smoothing paddle movement, adding more attributes to the brain. But in fact, it turns out that the game worse when i pass in more variables.

It turns out that lowering the alpha from 0.005 to 0.0001 does the trick !!! Check it out. You can save your model as well as loading the prebuilding model by click on "Load Model button"

The code is not that clean, abit messy because i was trying out stuff. Will refactor when i have time. This is cool !

[Try it](http://codericko.neocities.org/dqn/brickworld/v1.2/) ! Note, you can use developer console to check real-time between consecutive hits vs misses.

**Comment by gitricko** [comment]
Got [javascript-breakout](http://codericko.neocities.org/dqn/jsbreakout/) to work... Check it out !

**Comment by alperenkirimer** [comment]
@gitricko Hi we are also working on a deep q learning game using reinforcejs. Just wondering how many iterations do we need to see the ai acting clever?

**Comment by gitricko** [comment]
@alperenkirimer For me, it took a few hours because my environment is simplistics... It may take a day using CPU
